SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -24,6 +26,12 @@ void USpatialDispatcher::ProcessOps(Worker_OpList* OpList)
 	{
 		Worker_Op* Op = &OpList->ops[i];
 
+		if (OpsToSkip.Contains(Op))
```

Comment:
Two nits: 
1. do you think OpsToSkip should be a TSet? Contains for a set is O(1) but for Array it's O(n), though n is really small here so it probably doesn't matter. Mike thinks it should be an array so I'm fine either way.
2. Should we do a check before calling Contains to see if OpsToSkip is empty before doing a check? Might be a micro-optimization but I'm only concerned because this is a hot path.

Reply:
1.  For such a small startup-only use case, I think TArray is ok.  For larger data sets, totally agree.
2. Good call, i've made the change.